css/CSS2/linebox/vertical-align-top-bottom-001.html

```diff
@@ -0,0 +1,85 @@
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
+<style>
+section.test {
+  display: inline-block;
+  font-size: 20px;
+  line-height: 1.5;
+  font-family: Arial;
+  font-family: Ahem;
+}
+section.test > div {
+  background: blue;
+  margin-bottom: 1em;
+}
+.filler {
+  display: inline-block;
+  background: cyan;
+  height: 3em;
+  width: 1em;
+}
+.target {
+  display: inline-block;
+  background: orange;
+  width: 1em;
+  height: 1em;
+}
+div.top, section.top .target { vertical-align: top; }
+div.text-top, section.text-top .target { vertical-align: text-top; }
+div.text-bottom, section.text-bottom .target { vertical-align: text-bottom; }
+div.bottom, section.bottom .target { vertical-align: bottom; }
+.test .fail {
+  outline: red solid 5px;
+}
+</style>
+<body>
+<section class="test top">
+  <div><div class="filler"></div><div class="target" data-y="0"></div></div>
+  <div><div class="filler top"></div><div class="target" data-y="0"></div></div>
+  <div><div class="filler text-top"></div><div class="target" data-y="0"></div></div>
+  <div><div class="filler bottom"></div><div class="target" data-y="0"></div></div>
+  <div><div class="filler text-bottom"></div><div class="target" data-y="0"></div></div>
+</section>
+<section class="test text-top">
+  <div><div class="filler"></div><div class="target" data-y="44"></div></div>
+  <div><div class="filler top"></div><div class="target" data-y="5"></div></div>
+  <div><div class="filler text-top"></div><div class="target" data-y="5"></div></div>
+  <div><div class="filler bottom"></div><div class="target" data-y="35"></div></div>
+  <div><div class="filler text-bottom"></div><div class="target" data-y="40"></div></div>
+</section>
+<section class="test text-bottom">
+  <div><div class="filler"></div><div class="target" data-y="44"></div></div>
+  <div><div class="filler top"></div><div class="target" data-y="5"></div></div>
+  <div><div class="filler text-top"></div><div class="target" data-y="5"></div></div>
+  <div><div class="filler bottom"></div><div class="target" data-y="35"></div></div>
+  <div><div class="filler text-bottom"></div><div class="target" data-y="40"></div></div>
+</section>
+<section class="test bottom">
+  <div><div class="filler"></div><div class="target" data-y="49"></div></div>
+  <div><div class="filler top"></div><div class="target" data-y="40"></div></div>
+  <div><div class="filler text-top"></div><div class="target" data-y="45"></div></div>
+  <div><div class="filler bottom"></div><div class="target" data-y="40"></div></div>
+  <div><div class="filler text-bottom"></div><div class="target" data-y="45"></div></div>
+</section>
+<script>
+for (let target of document.getElementsByClassName('target')) {
+  let container = target.parentElement;
+  let filler = container.firstElementChild;
+  let section = container.parentElement;
+  let pass = false;
+  test(() => {
+    document.fonts.ready.then(()=> {
```

Comment:
This is the cause of the harness error in https://wpt.fyi/results/css/CSS2/linebox/vertical-align-top-bottom-001.html?label=pr_head&max-count=1&pr=18956, but more importantly it's not the same code as in https://chromium.googlesource.com/chromium/src/+/cf88186a260efa4875553eb99f79bbbff8d3130a/third_party/blink/web_tests/external/wpt/css/CSS2/linebox/vertical-align-top-bottom-001.html.

Reply:
@LukeZielinski this doesn't work because the test is sync and has already finished when the assert is run.

Reply:
Take 2, this time using explicit_done, let me know if that's still missing the mark.

As for the code being different - why is that an issue? I thought fixing broken imports here was common practice since the changes just get synced back anyway.